visualizations/line_chart/tests/test_line_chart.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+import pandas as pd
+import warnings
+from webviz_line_chart import LineChart
+
+line_mock_data = {
```

Comment:
Just nice to have if we want to write more tests for this component in the future.